llvm/test/CodeGen/AMDGPU/si-pre-allocate-wwm-regs.mir

```diff
@@ -0,0 +1,49 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 5
```

Comment:
Best to remote this and remove the comment then

Reply:
What is remote?